test/test_linalg.py

```diff
@@ -4004,6 +4005,9 @@ def check(equation, *operands, expected_output):
         check('...a->', [[2], [4]], expected_output=6)
         check('a...b->ab', [[[1], [2]], [[3], [4]]], expected_output=[[3], [7]])
 
+        # Capital letters OK
+        check('A', [1], expected_output=1)
```

Comment:
```suggestion
        check('A', [1], expected_output=[1])
```
Other than that looks good.